### DISCUSSION TITLE
List of Figures is showing unwanted information because of footnote

### DISCUSSION THREAD

**Discussion by irsimoes** [joaomlourenco/novathesis#468] [Q&A] [answered]
Hello! When trying to add a footnote to the caption of an image, I resort to the usage of /footnotemark and /footnotetext. It works, but it causes the List of Images to include a weird unformatted reference to it right after the correct reference.  The following picture illustrates what I am trying to explain better.

![image](https://github.com/joaomlourenco/novathesis/assets/71334178/52f72560-ff1b-4899-b2a4-dbc3dd44ae67)

 
Is there anything that can be done?
Thank you.

**Comment by joaomlourenco** [comment, marked as answer]
Use
```latex
\caption[Text for the list of figures, without the footnote stuff]{Text for the figure with the footnote stuff}
```

**Comment by irsimoes** [comment]
It worked, thank you!
